engine/index.js

```diff
@@ -462,3 +462,8 @@ export default {
   buildStatus,
   buildIndex,
 };
+
+const test = {
```

Comment:
The babel plugin doesn't work with babel 6+:
  https://github.com/speedskater/babel-plugin-rewire/issues/71

At this point I'm thinking that exporting the items we want to test via a non-default export makes the most sense.